### DISCUSSION TITLE
No option to export signature fields in export profile

### DISCUSSION THREAD

**Discussion by CAIDcreative** [solspace/craft-freeform#578] [Feature Requests]
Currently there's no way of exporting the signature field via the export profile. Understand it's stored in the database as a big long string that may be a bit awkward for general use but would still be nice to have an option of doing so if I want to.

**Craft & Plugin Info (please complete the following information):**

- Craft Version: Pro 4.3.3
- Freeform Version: 4.0.8
- Freeform Edition: Pro
- Fresh Install or Upgrade: Upgrade

**Comment by CAIDcreative** [comment]
Just an update on this - I think I've solved this myself by editing the vendor file vendor/solspace/craft-freeform/packages/plugin/src/Models/Pro/ExportProfileModel.php and simply commenting out the two OR conditions (on line 159 and 210) which reference "$field instanceof SignatureField" and is clearly meant to purposely skip this field.

I think there may be a legitimate reasons for doing this but not sure what it was: these lone fields increase the filesize of the export by quite a bit so could have been a filesize thing, also possibly just kept out for security reasons (like how credit card details are skipped).

At any rate this solves my problem for the moment.

**Comment by kjmartens** [comment]
Thanks @CAIDcreative, we'll make this change as well in Freeform 4. 🙂
